src/main/webapp/view/transcript/transcript.js

```diff
@@ -171,3 +178,34 @@ function scrollToTopOfTranscript(transcriptLine) {
   const ulElementOffset = transcriptLine.parentElement.offsetTop;
   transcriptContainer.scrollTop = transcriptLine.offsetTop - ulElementOffset;
 }
+
+/**
+ * Returns the next transcript line based on `currentTimeMs`.
+ */
+function getNextTranscriptLine(currentTimeMs) {
+  const nextTranscript = currentTranscriptLine.nextElementSibling;
+  // Video is playing normally.
+  if (isWithinCurrentTimeRange(currentTimeMs, nextTranscript)) {
+    return nextTranscript;
+  }
+  return findClosestTranscriptLine(currentTimeMs);
+}
+
+/**
+ * Searches for and returns the closest transcript line
+ * based on `currentTimeMs`.
+ */
+function findClosestTranscriptLine(currentTimeMs) {
+  let transcriptLinePointer = document.getElementsByTagName('li')[0];
+  while (transcriptLinePointer != null &&
+         !isWithinCurrentTimeRange(currentTimeMs)) {
+    transcriptLinePointer = transcriptLinePointer.nextElementSibling;
+  }
+  // This happens if the time does not fall in the time
+  // range of any transcript.
+  if (transcriptLinePointer === null) {
+    removeBold(currentTranscriptLine);
```

Comment:
This is an unexpected side affect to calling the function, which should be avoided.